gtk/src/default/gtk-3.20/_tweaks.scss

```diff
@@ -316,343 +316,27 @@ treeview {
 }
 
 
-// We want users to understand tabs easier
-// They know this classic styling from popular web-browsers
-
-notebook {
-  > header {
-    padding: 1px;
-    border-color: $borders_color;
-    border-width: 1px;
-
-    background: $bg_color;
-
-    &:backdrop {
-      border-color: $backdrop_borders_color;
-      background-color: $backdrop_bg_color;
-    }
-
-    tabs { margin: -1px; }
-
-    &.top {
-      border-style: none none solid none;
-      > tabs {
-        margin-bottom: -2px;
-        padding-top: 4px;
-        > tab {
-          border: 1px solid transparent;
-          border-radius: 4px 4px 0 0;
-
-          &:hover:not(:checked):not(:backdrop) {
-            background-color: transparentize($base_color, 0.5);
-            box-shadow: none;
-            border-color: $borders_color;
-          }
-
-          &:backdrop {
-            box-shadow: none;
-          }
-
-          &:checked {
-            background-color: $base_color;
-            border: 1px solid $borders_color;
-            border-bottom-color: transparent;
-            box-shadow: none;
-
-            &:hover { background-color: $base_color; }
-            &:backdrop { background-color: $backdrop_base_color; }
+// Make the tabs slightly less heavy
```

Comment:
We can remove or rephrase the comment, I think, since it is not really clear ("less heavy" respect what?)

Reply:
Indeed, sorry for the weird comment. I'll update it if you guys like the idea

Reply:
I still need to test the branch. I am positive, but since there's a poll, let's wait for the other replies

Reply:
What comment would you suggest btw? :smile_cat:

Reply:
None? :D